manim/utils/space_ops.py

```diff
@@ -766,3 +767,15 @@ def earclip_triangulation(verts: np.ndarray, ring_ends: list) -> list:
 
     meta_indices = earcut(verts[indices, :2], [len(indices)])
     return [indices[mi] for mi in meta_indices]
+
+
+def perpendicular_bisector(line: Sequence[Sequence[float]]):
```

Comment:
Can you rewrite this function to work in 3D? As there is no unique perpendicular bisector in 3D, you can specify an additional vector as a kwarg, which can be set to OUT or IN as default (so that the perpendicular bisector is perpendicular to both the direction vector of the line and this additional vector, so that the default behavior is to put the bisector vector on the xy-plane)

Reply:
This is going to be wayy harder to implement, but I'll try and see what I can do :)

Reply:
@Darylgolden your suggestion worked! 👍